File: spring-sms/src/main/java/com/example/springsms/domain/user/controller/UserController.java
package com.example.springsms.domain.user.controller;

import com.example.springsms.domain.user.controller.dto.SignupRequest;
import com.example.springsms.domain.user.controller.dto.UserSendMailRequest;
import com.example.springsms.domain.user.service.SendAuthCodeService;
import lombok.RequiredArgsConstructor;
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/user")
@RequiredArgsConstructor
public class UserController {
    private final SendAuthCodeService sendAuthCodeService;

    @PostMapping("/send")
    public void sendUserAuthCode(@RequestBody UserSendMailRequest request) {
        sendAuthCodeService.execute(request);
    }

    @PostMapping("/sign-up")
    public void signUp(@RequestBody SignupRequest request) {

    }
}

File: jpa-second-cache/src/main/kotlin/com/example/jpasecondcache/domain/user/presentation/UserController.kt
package com.example.jpasecondcache.domain.user.presentation

import com.example.jpasecondcache.domain.user.entity.User
import com.example.jpasecondcache.domain.user.usecase.QueryUsersUseCase
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RequestMapping("/users")
@RestController
class UserController(
    private val queryUsersUseCase: QueryUsersUseCase
) {

    @GetMapping
    fun getUsers(): List<User> {
        return queryUsersUseCase.execute()
    }
}